### PR TITLE
[docs/enhancement] - commaai/openpilot/README.md - tweaking a few things in comma ai open pilot README.md file (layout, ToC - Table of Contents, Grammar, etc.)

### PR DESCRIPTION
**Pull request by ghost**
## [docs/enhancement] - commaai/openpilot/README.md - tweaking a few things in comma ai open pilot readme file (layout, ToC - Table of Contents, Grammar, etc.)

**this PR does basically consist of minor tweaks within:**
- *improving overall document grammar*
- *improving overall document layout*
- *improving overall document style*
- *adding in a table of content to the document*
- *adding in logos of license and company to the document*

**the overall motivation behind such lies within:**
- *improving overall document quality*
- *improving overall document accessibility*
- *making the document easier to be read*
- *making the document more accessible to be read*

**Furthermore:**
- *helping to strike a better first impact whenever opening the document which can potentially*
 - *lead to more views of the repository*
 - *lead to more forks of the repository*
 - *lead to more commits to the repository*
 - *lead to more stars to the repository*

**Cheers**

</img> </img>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,12 +185,12 @@ To understand how the services interact, see `selfdrive/service_list.yaml`
...
 ------
 
-By default openpilot creates an account and includes a client for chffr, our dashcam app. We use your data to train better models and improve openpilot for everyone.
+By default, openpilot creates an account and includes a client for chffr, our dashcam app. We use your data to train better models and improve openpilot for everyone.
 
 It's open source software, so you are free to disable it if you wish.
 
-It logs the road facing camera, CAN, GPS, IMU, magnetometer, thermal sensors, crashes, and operating system logs.
-The user facing camera is only logged if you explicitly opt-in in settings.
+It logs the road facing the camera, CAN, GPS, IMU, magnetometer, thermal sensors, crashes, and operating system logs.
```
Comment (rbiasini) [inline README.md]: road-facing camera?
Other than that it looks good. Thanks

Comment (tem44) [reply README.md]: revised. waiting for additional feedback. looking forward

**Review by rbiasini — COMMENTED** [review]
It logs the road-facing camera. Without `the`. Right? 🙂 Like user-facing camera below.

**Comment by tem44** [comment]
done. fixed

File: README.md [hunk 2]
```diff
@@ -130,8 +147,10 @@ If your car has adaptive cruise control and lane keep assist, you are in luck. U
 We've written guides for [Brand](https://medium.com/@comma_ai/how-to-write-a-car-port-for-openpilot-7ce0785eda84) and [Model](https://medium.com/@comma_ai/openpilot-port-guide-for-toyota-models-e5467f4b5fe6) ports. These guides might help you after you have the basics figured out.
 
 - BMW, Audi, Volvo, and Mercedes all use [FlexRay](https://en.wikipedia.org/wiki/FlexRay) and are unlikely to be supported any time soon.
-- We put time into a Ford port, but the steering has a 10 second cutout limitation that makes it unusable.
-- The 2016-2017 Honda Accord use a custom signaling protocol for steering that's unlikely to ever be upstreamed.
+- We put time into a Ford port, but the steering has a 10-second cutout limitation that makes it unusable.
+- The 2016-2017 Honda Accord uses a custom signaling protocol for steering that's unlikely to ever be streamed.
```
Comment (energee) [inline README.md]: upstreamed?
```suggestion
- The 2016-2017 Honda Accord uses a custom signaling protocol for steering that's unlikely to ever be upstreamed.
```

Comment (tem44) [reply README.md]: fixed

---

Shaggy: I can already taste those chocolate-covered hotdogs.
Velma: Yuck! His stomach must be made of scrap iron.
Shaggy: Can I help it if my first toy was a garbage disposal?

from scooby-doo

**Comment by tem44** [comment]
dear gentleman, please feel free to spot in any further feedback. cheers

**Comment by rbiasini** [comment]
it had conflicts with our internal version, so we cherry picked the changes and it will be in next release. Thansk.

**Pull request closed without merging**